recipes/poliastro/build.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+pip install --no-deps .
```

Comment:
Got it, I will make the change then. For reference: https://github.com/conda-forge/cartopy-feedstock/blob/a23c28a/recipe/build.sh#L5